src/network/networkd-link.c

```diff
@@ -1325,6 +1325,11 @@ static int link_set_bridge(Link *link) {
                 if (r < 0)
                         return log_link_error_errno(link, r, "Could not append IFLA_BRPORT_COST attribute: %m");
         }
+        if (link->network->priority != 0) {
```

Comment:
If I am reading the kernel source code right, the default priority is 32, so I could set 32 as the default value in the gperf . Then change this condition and only set priority when it's not 32? Or e.g. always set it? (But then default priority definition effectively moves from kernel to systemd)